rpcserver.go

```diff
@@ -6515,6 +6513,100 @@ func newRPCServer(listenAddrs []string, generator *BlkTmplGenerator, s *server)
 	return &rpc, nil
 }
 
+// handleBlockchainNotification callback for notifications from blockchain.
+// It notifies clients that are long polling for changes or subscribed to
+// websockets notifications.
+func (s *rpcServer) handleBlockchainNotification(notification *blockchain.Notification) {
```

Comment:
Why is this being added?  Isn't it already covered by `handleBlockchainNotification` on `blockManager`?